Controller/SnippetAreaController.php

```diff
@@ -72,7 +72,7 @@ public function getAction(Request $request, string $area): Response
         $webspaceKey = $webspace->getKey();
         $locale = $request->getLocale();
 
-        $includeExtension = $this->getBooleanRequestParameter($request, 'includeExtension', true, false);
+        $includeExtension = $this->getBooleanRequestParameter($request, 'includeExtension', false, false);
```

Comment:
was incnsistent to the [NavigationController](https://github.com/sulu/SuluHeadlessBundle/blob/83562fa5e9f661642eb24007cedaa89acf05106d/Controller/NavigationController.php#L66-L68)

Reply:
maybe we should add this to the `UPGRADE.md`?

Reply:
this is just that it accept `includeExtension=` without a value this is not a default value the last parameter is the default value which was not changed.